tests/test_setups/test_setup_functions.py

```diff
@@ -149,4 +149,14 @@ def test_setuplist_offset(self):
 
         all = set(setups.keys()).union(setups2.keys())
 
-        self.assertEqual(len(all), size * 2)
\ No newline at end of file
+        self.assertEqual(len(all), size * 2)
+
+    def test_get_cached_setup(self):
+        openml.config.set_cache_directory(self.static_cache_dir)
+        openml.setups.functions._get_cached_setup(1)
```

Comment:
It would be great to have a test which checks that there's an exception for trying to access an uncached dataset.

Reply:
Don't see how this would be different from test_get_uncached_setup() ?

We now test the following cases:
* test_get_setup (downloads a setup and caches it)
* test_get_cached_setup (retrieves a setup from cache)
* test_get_uncached_setup (tries to retrieve an uncached setup, and throws an error)